plugin.go

```diff
@@ -24,15 +25,21 @@ var (
 )
 
 type Plugin struct {
-	ClusterConfig types.ClusterConfig
-	ValueFiles    []string
-	KubeConfig    string
-	KubeContext   string
-	Dryrun        bool
-	Diffrun       bool
+	ClusterConfig     types.ClusterConfig
+	ClusterConfigPath string
+	ClustersList      report.Clusters
+	ValueFiles        []string
+	KubeConfig        string
+	KubeContext       string
+	Dryrun            bool
+	Diffrun           bool
+	Audit             bool
+	AuditFile         string
+
 }
 
 func (p *Plugin) Exec() error {
+	if ! p.Audit {
 	// Init Kubernetes config
```

Comment:
With the new `if`-block, the contents should be indented. Perhaps run `go fmt` to get consistent formatting?